crates/encoding/src/estimate.rs

```diff
@@ -226,8 +231,8 @@ impl BumpEstimator {
 fn estimate_arc_lines(scaled_stroke_width: f32) -> (u32, f32) {
     // These constants need to be kept consistent with the definitions in `flatten_arc` in
     // flatten.wgsl.
-    const MIN_THETA: f32 = 1e-4;
-    const TOL: f32 = 0.1;
+    const MIN_THETA: f32 = 1e-6;
+    const TOL: f32 = 0.25;
```

Comment:
A config uniform for tolerances might be a good idea. It may also make sense to have a common set of constant declarations that are shared by cpu_shaders and the estimate code (though they live in different crates).